src/components/Router/components/CustomRoute/index.tsx

```diff
@@ -13,7 +14,14 @@ function CustomRoute({ path, isPrivate, ...props }: RouteProps & CustomRouteProp
   const userAuthenticated = LocalStorageService.getValue(LOCAL_STORAGE_KEYS.session);
 
   if (userAuthenticated) {
-    return isPrivate ? <Route path={path} {...props} /> : <Redirect to={PATHS.home} />;
+    return isPrivate ? (
+      <Fragment>
+        <Navbar />
+        <Route path={path} {...props} />
```

Comment:
Is there a dashboard component? those routes could be added too `Dashboard` which has another Switch and routes there that acts like private routes. If not it is ok for me rn

Reply:
It doesn't exist atm, but it can be added :ok_hand: